src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/UserProvider/LdapFactory.php

```diff
@@ -40,6 +41,7 @@ public function create(ContainerBuilder $container, string $id, array $config)
             ->replaceArgument(6, $config['filter'])
             ->replaceArgument(7, $config['password_attribute'])
             ->replaceArgument(8, $config['extra_fields'])
+            ->replaceArgument(9, $roleFetcher)
```

Comment:
```suggestion
            ->replaceArgument(9, $$config['role_fetcher'] ? new Reference($config['role_fetcher']) : null)
```

I think we can keep it in one line

Reply:
Indeed. Comment addressed.